src/Caching/Storages/SQLiteStorage.php

```diff
@@ -65,6 +65,31 @@ public function read($key)
 
 
 	/**
+	 * Read from cache.
+	 * @param  string key
+	 * @return array key => value pairs, missing items are omitted
+	 */
+	public function bulkRead(array $keys)
+	{
+		$stmt = $this->pdo->prepare('SELECT key, data, slide FROM cache WHERE key IN (?' . str_repeat(',?', count($keys) - 1) . ') AND (expire IS NULL OR expire >= ?)');
+		$stmt->execute(array_merge($keys, [time()]));
```

Comment:
Replace with `$stmt->execute(time(), ...$keys)` + update conditions order in SQL